CONTRIBUTING.md

```diff
@@ -108,7 +108,7 @@ executable will be in your current directory and can be run as shown in the
 
 + Run the test suite locally:
 
-  `go generate ./pkg/... && make check test testrace`
+  `make generate && make check test testrace`
```

Comment:
I disagree - this stanza is explicitly described as "run the test suite locally". That implies that all tests will be run. Without running the UI tests, not all of the tests will be run.